src/providers/toast.service.ts

```diff
@@ -4,6 +4,10 @@ import { ToastController } from 'ionic-angular';
 @Injectable()
 export class ToastService {
     private faveToast;
+    private noLocationToast;
```

Comment:
I suppose the original creator of this class made the same mistake, but there's no reason to have class-wide declarations at the top.  They're never really _used_, only assigned and immediately `present()`ed.